extensions/sharing/pkg/revaconfig/config.go

```diff
@@ -52,7 +52,7 @@ func SharingConfigFromStruct(cfg *config.Config) map[string]interface{} {
 							"provider_addr":       cfg.UserSharingDrivers.CS3.ProviderAddr,
 							"service_user_id":     cfg.UserSharingDrivers.CS3.ServiceUserID,
 							"service_user_idp":    cfg.UserSharingDrivers.CS3.ServiceUserIDP,
-							"machine_auth_apikey": cfg.UserSharingDrivers.CS3.MachineAuthAPIKey,
+							"system_user_api_key": cfg.UserSharingDrivers.CS3.SystemUserAPIKey,
```

Comment:
```suggestion
							"machine_auth_apikey": cfg.UserSharingDrivers.CS3.SystemUserAPIKey,
```
`machine_auth_apikey` is expected by the reva cs3 driver